CHANGELOG.md

```diff
@@ -92,24 +93,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Deprecated
 - CLI
-  - `--domain=...` is superseded by `--san=dNSName:...` and will be removed in a future release 
+  - `--domain=...` is superseded by `--san=dNSName:...` and will be removed in a future release
 
 ### Removed
 - API
   - Removed redirection from legacy frontend names (`members` -> `gov`, `nodes` -> `node`, `users` -> `app`) (#1543)
   - Removed old `install()` API, replaced by `make_endpoint()` in [0.11.1](https://github.com/microsoft/CCF/releases/tag/ccf-0.11.1) (#1541)
-    
+
 ## [0.12.2]
 ### Fixed
 - Fix published containers
-    
+
 ## [0.12.1]
 ### Changed
 - Release tarball replaced by a .deb
 
 ### Fixed
 - Fix LVI build for applications using CCF (#1466)
-    
+
```

Comment:
I would prefer we avoid reformatting unless we have an automatic formatter, such as markdownlint.

Reply:
https://github.com/DavidAnson/markdownlint looks like a plausible choice.